README.md

```diff
@@ -1,7 +1,16 @@
 # quantum-attendance
 An attendance program for any type of martial arts dojo.
 
-# Set up for local development
+# Set up for local development (with Docker)
+
+ 1. Install Docker
+ 2. run `docker-compose up`
+
+The first run might take a while due to dependencies being installed.
+In case dependencies change and the docker image needs to be rebuilt, just use 
+`docker-compose up --build`.
```

Comment:
Hrm... there used to be an issue with docker on mac and intermediate containers that would sit around and end up taking up all 64 gigs of hard drive space. It looks like they fixed that issue.